### ISSUE TITLE
Types are not working correctly

### ISSUE THREAD

**Issue by juancjara** [primer/components#534]
I'm using typescript on my app,  I added a basic `<Box>` with padding prop `p={3}` but got this error:

```
Type '{ children: Element; p: number; }' is not assignable to type 'IntrinsicAttributes & BoxProps & { children?: ReactNode; }'.
  Property 'p' does not exist on type 'IntrinsicAttributes & BoxProps & { children?: ReactNode; }'.ts(2322)
```
Reproduce link

[https://codesandbox.io/s/festive-shockley-knjb6](https://codesandbox.io/s/festive-shockley-knjb6)

**Comment by dmarcey** [comment]
I think that the issue here is this line from index.d.ts:

```  interface CommonProps extends BaseProps, StyledSystemProps.color, StyledSystemProps.space {}```

should instead be

```   interface CommonProps extends BaseProps, StyledSystem.ColorProps, StyledSystem.SpaceProps {} ```

I can verify and get out a PR.

**Comment by emplums** [comment]
Fixed in 13.5.0!
